src/Application/Commands/DeleteShoppingList/DeleteShoppingListCommand.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Lindyhopchris\ShoppingList\Application\Commands\DeleteShoppingList;
+
+use Lindyhopchris\ShoppingList\Persistance\ShoppingListRepositoryInterface;
+
+class DeleteShoppingListCommand implements DeleteShoppingListCommandInterface
+{
+    /**
+     * @var ShoppingListRepositoryInterface
+     */
+    private ShoppingListRepositoryInterface $repository;
+
+    /**
+     * @param ShoppingListRepositoryInterface $repository
+     */
+    public function __construct(ShoppingListRepositoryInterface $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    public function execute(DeleteShoppingListModel $model): void
+    {
+        $list = $this->repository->findOrFail(
```

Comment:
Comment: this morning we talked about whether it was necessary to load the entity from storage to then delete it.

Reply:
Okay, I updated this file using the fileHandler. Once the fileHandler has the delete/unlink method in it then I could use it to delete the file with the given slug. For right now the updated code in this file is seeing if the file with the given slug exists.